ui/karma.conf.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(config) {
       'spec/spec-helper.js': ['webpack', 'sourcemap'],
       'spec/index.js': ['webpack', 'sourcemap'],
     },
-    reporters: ['dots'],
+    reporters: ['dots'], // can also use 'verbose' and/or 'progress' for more detailed reporting
```

Comment:
And @timraymond, `--verbose` does work but means verbose mode for the karma process lifecycle, not the tests themselves. I'm looking for detailed reporting, which would normally work via `--reporters=verbose` but doesn't seem to be currently.

@cryptoquick suggested `npm test -- --reporters=verbose` -- that turns out to work!